interface/orders/procedure_stats.php

```diff
@@ -331,119 +331,117 @@ function process_result_code($row)
+            }
+            ?>
+            </select>
+        </div>
+    </div>
+</div>
+<br>
+<div class="col-8 col-md-8">
+    <div class="row">
+        <div class="col">
+            <?php echo xlt('Facility'); ?>:
+            <?php dropdown_facility($form_facility, 'form_facility', false); ?>
+        </div>
+        <div class="col">
+            <?php echo xlt('From'); ?>
+            <input type='text' class='form-control datepicker' name='form_from_date' id='form_from_date' size='10' value='<?php echo attr(oeFormatShortDate($from_date)); ?>' />
+            </div>
+            <div class="col">
+            <?php echo xlt('To{{Range}}'); ?>
+        <input type='text' class='form-control datepicker' name='form_to_date' id='form_to_date' size='10' value='<?php echo attr(oeFormatShortDate($to_date)); ?>' />
+        </div>
+    </div>
+</div>
+<br>
+<div class="col-8 col-md-8">
+    <div class="row">
+        <div class="col">
+            <?php echo xlt('Columns'); ?>:
+            <select class='form-control' name='form_show[]' size='4' multiple title='<?php echo xla('Hold down Ctrl to select multiple items'); ?>'>
+                <?php
+                foreach ($arr_show as $key => $value) {
+                    $title = $value['title'];
+                    if (empty($title) || $key == 'title') {
+                        $title = $value['description'];
+                    }
+
+                    echo "    <option value='" . attr($key) . "'";
+                    if (is_array($form_show) && in_array($key, $form_show)) {
+                        echo " selected";
+                    }
+
+                    echo ">" . text($title) . "</option>\n";
+                }
+                ?>
+            </select>
+        </div>
+    </div>
+</div>
+<br>
+<div class="col-8 col-md-8">
+    <div class="row">
+        <div class="col">
+            <?php echo xlt('To{{Destination}}'); ?>:
+
+            <?php
+            foreach (array(1 => xl('Screen'), 2 => xl('Printer'), 3 => xl('Export File')) as $key => $value) {
+                echo "   <input type='radio' name='form_output' value='" . attr($key) . "'";
+                if ($key == $form_output) {
+                    echo ' checked';
+                }
 
-        echo " />" . text($value) . " &nbsp;";
-    }
-    ?>
-</td>
-<td align='right' valign='top' class='detail' nowrap>
-<input type='submit' class='btn btn-primary' name='form_submit' value='<?php echo xla('Submit'); ?>' title='<?php echo xla('Click to generate the report'); ?>' />
-</td>
-</tr>
-<tr>
-<td colspan='5' height="1">
-</td>
-</tr>
-</table>
-    <?php
+                echo " />" . text($value) . " &nbsp;";
+            }
+            ?>
+        </div>
+    </div>
+</div>
+<br>
+<div class="col-4 col-md-4">
+    <div class="col">
+         <input type='submit' class='btn btn-primary' name='form_submit' value='<?php echo xla('Submit'); ?>' title='<?php echo xla('Click to generate the report'); ?>' />
```

Comment:
submit button need add font awesome icon.

Reply:
Okay

Reply:
Hi @adad20 

Check out FontAwesome.scss this file, This file contains OpenEMR common icons. Use `btn-save` will suitable in this button.

Reply:
hi @stu01509 Then should I add text too in the button or the icon only?

Reply:
hi @adad20 , Just add btn-save to the class:
```
btn btn-primary btn-save
```
And then test it out (it places a nice icon on the button along with the text that is already there)